nrpy/infrastructures/CarpetX/general_relativity/T4DD_to_T4UU.py

```diff
@@ -0,0 +1,157 @@
+"""
+Generates function to raise the stress-energy tensor using the BSSN variables.
+
+Authors: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+         Samuel Cupp
+"""
+
+from typing import Union, cast, List
+from inspect import currentframe as cfr
+from types import FrameType as FT
+import sympy
```

Comment:
please use `import sympy as sp`, for consistency across codebase.

Reply:
changed